### DISCUSSION TITLE
Date to find two archives with same name

### DISCUSSION THREAD

**Discussion by jcf278** [Flow-Launcher/Flow.Launcher#1648] [Q&A]
Hi everyone.

Is there a way to add a date to search two or more archives with the same name?

The example bellow is about a file I have on two different locations, In one location is newer, but I cannot recognize wich is the right file to select in the bar.

![image](https://user-images.githubusercontent.com/56937020/207656437-e87f9afa-b856-4da5-abee-53fde65c2690.png)

**Comment by onesounds** [comment]
1. we added full location with preview panel in 1.10. press f1 or use "always preview" in general tap

2. tooltop will showing fullpath while mouse hover

3. I have plan to adding more info in preview panel. it will be add in next time.

**Reply by jcf278** [reply]
thanks for reply.. In my case this is not helpful because I don't know wich location is the correct, I choose according the date of last modification. I think will be usefull if add the date after the name, like this.

![image](https://user-images.githubusercontent.com/56937020/207912870-6b00d185-cce1-4468-8c34-8df6353704bf.png)

thanks for the attention

**Reply by deefrawley** [reply]
If you are using Everything then you can have the search results ordered by the date created or date modified.
Go into Settings - Plugin - Explorer Plugin - Everything Settings and chnage the Sort Order.
